release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -9,10 +10,10 @@
 import os
 
 # global variables for the cluster informations
-cluster_id = str(uuid.uuid4()).split("-")[0]
-ray_cluster_name = "cluster-" + cluster_id
-ray_service_name = "service-" + cluster_id
-locust_id = "ray-locust-" + cluster_id
+cluster_id = None
```

Comment:
Could you make the global variables all-caps (e.g. `cluster_id` -> `CLUSTER_ID`)?